ocaml/gencert/gencert.ml

```diff
@@ -51,11 +68,17 @@ let () =
   (* if necessary use Unix.localtime to debug *)
   D.debug "%s" dbg ;
   match Sys.argv with
-  | [|_; path|] when Sys.file_exists path ->
+  | [|_; path; _|] when Sys.file_exists path ->
       D.info "file already exists at path (%s) - doing nothing" path ;
       exit 0
-  | [|_; path|] ->
-      main ~dbg ~path
+  | [|_; path; sni|] -> (
+    match sni_of_string sni with
+    | Some sni ->
+        main ~dbg ~path ~sni
+    | None ->
+        D.error "SNI must be default or xapi:pool" ;
+        exit 1
+  )
   | _ ->
-      D.error "Usage: %s PATH" program_name ;
+      D.error "Usage: %s PATH [default|xapi:pool]" program_name ;
```

Comment:
I think `sni` is required, so according to 'docopt' the usage string should use round brackets `(default|xapi:pool)`